src/integrationTest/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceIT.java

```diff
@@ -126,30 +127,135 @@ void testDeleteHearingRequest_WithAllMandatoryFields() {
     @Sql(scripts = {DELETE_HEARING_DATA_SCRIPT, INSERT_CASE_HEARING_DATA_SCRIPT})
     void testDeleteHearingRequest_WithNullHearingId() {
         DeleteHearingRequest request = TestingUtil.deleteHearingRequest();
-        Exception exception = assertThrows(BadRequestException.class, () -> {
-            hearingManagementService.deleteHearingRequest(null, request);
-        });
+        Exception exception = assertThrows(BadRequestException.class, () -> hearingManagementService
+            .deleteHearingRequest(null, request));
         assertEquals("Invalid hearing Id", exception.getMessage());
     }
 
     @Test
     @Sql(scripts = {DELETE_HEARING_DATA_SCRIPT, INSERT_CASE_HEARING_DATA_SCRIPT})
     void testDeleteHearingRequest_WithHearingId_NotPresentInDB() {
         DeleteHearingRequest request = TestingUtil.deleteHearingRequest();
-        Exception exception = assertThrows(HearingNotFoundException.class, () -> {
-            hearingManagementService.deleteHearingRequest(2000000001L, request);
-        });
+        Exception exception = assertThrows(HearingNotFoundException.class, () -> hearingManagementService
+            .deleteHearingRequest(2000000001L, request));
         assertEquals("No hearing found for reference: 2000000001", exception.getMessage());
     }
 
     @Test
     @Sql(scripts = {DELETE_HEARING_DATA_SCRIPT, INSERT_CASE_HEARING_DATA_SCRIPT})
     void testDeleteHearingRequest_WithInvalidHearingIdFormat() {
         DeleteHearingRequest request = TestingUtil.deleteHearingRequest();
-        Exception exception = assertThrows(BadRequestException.class, () -> {
-            hearingManagementService.deleteHearingRequest(300000000L, request);
-        });
+        Exception exception = assertThrows(BadRequestException.class, () -> hearingManagementService
+            .deleteHearingRequest(300000000L, request));
         assertEquals("Invalid hearing Id", exception.getMessage());
     }
 
+    @Test
```

Comment:
for all the new tests can we either add an assert for the response or verify call to ensure a subsequent calls were made